pyproject.toml

```diff
@@ -17,6 +17,10 @@ redisearch = "^2.1.1"
 feedgen = "^0.9.0"
 PyGObject = "^3.42.0"
 watchgod = "^0.7"
+FastAPI-SQLAlchemy = "^0.2.1"
+alembic = "^1.7.5"
+python-dotenv = "^0.19.2"
+psycopg2-binary = "^2.9.3"
```

Comment:
Are we happy that all these are the right version constraints (and all needed?)  If we move from a `.env` file to using the settings from the FastAPI app, do we get to drop `python-dotenv` ?